source/_docs/automation/trigger.markdown

```diff
@@ -868,8 +868,7 @@ See the [Persistent Notification](/integrations/persistent_notification/) integr
 
 ## Webhook trigger
 
-Webhook trigger fires when a web request is made to the webhook endpoint: `/api/webhook/<webhook_id>`. The webhook endpoint is created automatically when you set it as the `webhook_id` in an automation trigger.
-
+Webhook trigger fires when a web request is made to the webhook endpoint: `/api/webhook/<webhook_id>`. The webhook endpoint is created automatically when you set it as the `webhook_id` in an automation trigger. You may need to restart Home Assistant if you are setting up a webhook for the first time.
```

Comment:
This not correct, as restant isn't needed.

Ik you run into a situation where in is, hou might have ran into a bug, which should be reported with reproduction steps instead.